### PR TITLE
2 new Wallpaper

### PR DESCRIPTION
**Pull request by nikurasutan**
I added 2 more wallpaper for more moe. The huge changes in the Moeification.pot are there because I opend the file in Poedit and saved it.

### CODE REVIEW

File: Moeification.sh [hunk 1]
```diff
@@ -273,6 +275,20 @@ if [ ! $? -eq 0 ]; then
...
     echo "$(eval_gettext "Downloaded the Your Name 2 wallpaper.")"
 fi
+wget https://i.redd.it/02c3t2v8zl561.png -O "/usr/share/weebpapers/(Moe) Yorimoi.png" > /dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+    echo "$(eval_gettext "Oh no, an error occured downloading the Yoimoi wallpaper, the girls will never reach antarctica!") $(eval_gettext "Aborting now.")"
+    exit 1
+else
+    echo "$(eval_gettext "Downloaded the Yorimoi wallpaper.")"
+fi
+wget https://teddit.net/pics/w:null_35a1utot7vh61.png -O "/usr/share/weebpapers/(Moe) Ai Ohto.png" > /dev/null 2>&1
```
Comment (dominichayesferen) [inline Moeification.sh]: The link died.

File: Moeification.sh [hunk 2]
```diff
@@ -273,6 +275,20 @@ if [ ! $? -eq 0 ]; then
 else
     echo "$(eval_gettext "Downloaded the Your Name 2 wallpaper.")"
 fi
+wget https://i.redd.it/02c3t2v8zl561.png -O "/usr/share/weebpapers/(Moe) Yorimoi.png" > /dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+    echo "$(eval_gettext "Oh no, an error occured downloading the Yoimoi wallpaper, the girls will never reach antarctica!") $(eval_gettext "Aborting now.")"
```
Comment (dominichayesferen) [inline Moeification.sh]: Just FYI I'd do this myself post-merge, but... I'd personally change 'antarctica' to 'Antarctica', and change it from 'wallpaper, the' to 'wallpaper. The'.

**Review by dominichayesferen — CHANGES_REQUESTED** [review]
There's only one breakage that means I'm gonna have to mark this as NEEDS CHANGES, but this is otherwise a GREAT first merge request, well done!

I'll probably end up regenerating the .pot in case again after merge anyway, so... yeah, dw, hopefully it still works.

**Comment by nikurasutan** [comment]
So, I fixed the Links and the Typos now. Sorry for not replying for so long, I had much things to do like my finals. Now I finally found time to fix all the things you mentioned. :-)

**Comment by dominichayesferen** [comment]
seems 👌🏻 now, so...

**Pull request merged**